src/Application.vala

```diff
@@ -32,6 +32,24 @@ public class PantheonCalculator.Application : Gtk.Application {
     public override void startup () {
         base.startup ();
 
+        var granite_settings = Granite.Settings.get_default ();
+        var gtk_settings = Gtk.Settings.get_default ();
+        gtk_settings.gtk_icon_theme_name = "elementary";
+
+        if (!gtk_settings.gtk_theme_name.has_prefix ("io.elementary")) {
+            gtk_settings.gtk_theme_name = "io.elementary.stylesheet.blueberry";
+        }
+
+        gtk_settings.gtk_application_prefer_dark_theme = (
+            granite_settings.prefers_color_scheme == Granite.Settings.ColorScheme.DARK
+        );
```

Comment:
minor nitpick, but the parenthesis here are unnecessary.

Reply:
I think it looks a little nicer the other way, but I'm not picky! fixed :)